src/main/java/ee/bitweb/core/retrofit/logging/writers/RetrofitLogLoggerWriterAdapter.java

```diff
@@ -0,0 +1,51 @@
+package ee.bitweb.core.retrofit.logging.writers;
+
+import ee.bitweb.core.retrofit.logging.RetrofitLoggingInterceptorImplementation;
+import ee.bitweb.core.retrofit.logging.mappers.RetrofitRequestMethodMapper;
+import ee.bitweb.core.retrofit.logging.mappers.RetrofitRequestUrlMapper;
+import ee.bitweb.core.retrofit.logging.mappers.RetrofitResponseBodySizeMapper;
+import ee.bitweb.core.retrofit.logging.mappers.RetrofitResponseStatusCodeMapper;
+import lombok.RequiredArgsConstructor;
+import org.slf4j.MDC;
+
+import java.util.Map;
+
+@RequiredArgsConstructor
+public class RetrofitLogLoggerWriterAdapter implements RetrofitLogWriteAdapter {
+
+    public static final String LOGGER_NAME = "RetrofitLogger";
+
+    private static final org.slf4j.Logger log = org.slf4j.LoggerFactory.getLogger(LOGGER_NAME);
+
+    @Override
+    public void write(Map<String, String> container) {
+        Map<String, String> currentContext = MDC.getCopyOfContextMap();
+
+        log(container);
```

Comment:
I would wrap this in try {} finally {} block so that MDC wouldn't be left unswapped